apps/ledger-live-desktop/src/newArch/features/DynamicContent/components/PortfolioContentCards/Slide.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { memo } from "react";
+import { useHistory } from "react-router-dom";
+
+import { PortfolioContentCard as Card } from "@ledgerhq/react-ui";
+import { openURL } from "~/renderer/linking";
+import type { PortfolioContentCard } from "~/types/dynamicContent";
+import type { CarouselActions } from "../../types";
+import LogContentCardWrapper from "../LogContentCardWrapper";
+
+export default memo(Slide);
+
+type Props = PortfolioContentCard & CarouselActions & { index: number };
+
+function Slide({ logSlideClick, dismissCard, index, ...card }: Props) {
```

Comment:
while card props spread, why you access to its props via card (in `handleClick` function and the return block ?

Reply:
Hey are you asking why am I not destructuring `card` ?

Reply:
yes

Reply:
Yeah I ended up destructuring it